ansible/roles/etcd/tasks/install.yml

```diff
@@ -0,0 +1,18 @@
+---
+- name: Install etcd
+  action: "{{ ansible_pkg_mgr }}"
+  args:
+        name: etcd
+        state: latest
+  notify:
+    - restart etcd
+  when: not is_atomic and ansible_distribution != "Ubuntu" and not is_coreos
```

Comment:
not saying you need to fix it here, but it seems like this test should be somehow just a `when: etcd_source_type == "package-manager"`     If you ever stumble into this code again.